edge-agent/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.6.0-linux-arm32v7
+ARG base_tag=1.0.6.2-linux-arm32v7
```

Comment:
who/which branch uses 1.0.6.1? that's why we use 1.0.6.2 here, right?

Reply:
1.0.6.1 was used for TRC, TestAnalyzer, NetworkController, & TwinTester. Those modules were added with non-root user recently.

Reply:
Please discuss with Varun for long term schema versioning.